server/grpc/api/service.go

```diff
@@ -1,53 +1,53 @@
-package newcore
+package api
 
 import (
 	"github.com/mesg-foundation/core/protobuf/definition"
 	"github.com/mesg-foundation/core/service"
 )
 
-func fromProtoService(s *definition.Service) *service.Service {
+func FromProtoService(s *definition.Service) *service.Service {
```

Comment:
let's expose only this one, `fromProtoTasks`, `fromProtoEvents`... don't need to be public, they are internal functions only

Reply:
done